app/views/course/statistics/aggregate/submission_due.json.jbuilder

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+json.name @all_students.first.name
+
+json.assessments @assessments do |assessment|
+  student_id = @all_students.first.id
```

Comment:
oh ya sorry about this. this can be put also outside the do-loop, and might be saving the execution time a bit.

Even though the effect might not be significant, but I moved this out from the loop